apps/frontend/tests/unit/Checklist.spec.ts

```diff
@@ -0,0 +1,85 @@
+import {FilteredDataModule} from '@/store/data_filters';
+import {InspecDataModule} from '@/store/data_store';
+import {SearchEntry, SearchModule} from '@/store/search';
+import Checklist from '@/views/Checklist.vue';
+import {ChecklistVuln} from '@mitre/hdf-converters';
+import {shallowMount, Wrapper} from '@vue/test-utils';
+import {readFileSync} from 'fs';
+import 'jest';
+import Vue from 'vue';
+import Vuetify from 'vuetify';
+import {loadChecklistFile} from '../util/testingUtils';
+
+const $router = {
+  currentRoute: {
+    path: '/checklists'
+  }
+};
+const vuetify = new Vuetify();
+
+const wrapper: Wrapper<Vue> = shallowMount(Checklist, {
+  vuetify,
+  mocks: {
+    $router
+  },
+  propsData: {}
+});
+
+describe('Datatable', () => {
```

Comment:
when the checklist mapper pr is merged, make sure we have test cases to handle the single istig + multiple istig in one file + istigs (single/multiple) from many files